siliconcompiler/sphinx_ext/dynamicgen.py

```diff
@@ -615,7 +615,7 @@ def document_task(self, path, toolmodule, taskmodule, taskname, toolname):
         # Annotate the target used for default values
         if chip.valid('option', 'target') and chip.get('option', 'target'):
             p = docutils.nodes.inline('')
-            target = chip.get('option', 'target')
+            target = chip.get('option', 'target').split('.')[-1]
```

Comment:
Why was this required?

Reply:
The target gets recorded as the python module name, but the reference just needs the module name